.gitignore

```diff
@@ -0,0 +1,59 @@
+# Prerequisites
```

Comment:
The `.gitignore` has never been useful to me and this seems unrelated to your patch. Could you please remove it?

I consider .gitignore a bad practise, because developers are expected to add the files they are interested in. They shouldn't blindly use `git add *`.